.github/workflows/release-app.yml

```diff
@@ -17,23 +20,47 @@ jobs:
       - id: get-matrix
         run: echo "nextcloud_versions=$(./get-matrix.sh)" | tee -a "$GITHUB_OUTPUT"
 
+  build-devcontainer:
```

Comment:
This build the devcontainer like described in the devcontainer.json definition. But this is just the build, so lifecycle-hooks like postCreate are not called here.